plugins/node/opentelemetry-instrumentation-koa/examples/src/tracer.ts

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const { KoaInstrumentation } = require('@opentelemetry/instrumentation-koa');
+// import { KoaInstrumentation } from '@opentelemetry/instrumentation-koa';
+// const { HttpInstrumentation } = require('@opentelemetry/instrumentation-http');
+import { HttpInstrumentation } from '@opentelemetry/instrumentation-http';
+
+import * as api from '@opentelemetry/api';
+import { registerInstrumentations } from '@opentelemetry/instrumentation';
+import { NodeTracerProvider } from '@opentelemetry/sdk-trace-node';
+import { SimpleSpanProcessor } from '@opentelemetry/sdk-trace-base';
+import { JaegerExporter } from '@opentelemetry/exporter-jaeger';
+import { ZipkinExporter } from '@opentelemetry/exporter-zipkin';
+import { Resource } from '@opentelemetry/resources';
+import { SemanticResourceAttributes } from '@opentelemetry/semantic-conventions'
+
+const EXPORTER = process.env.EXPORTER || '';
+
+export const setupTracing = (serviceName: string) => {
+  const provider = new NodeTracerProvider({
+    resource: new Resource({
+      [SemanticResourceAttributes.SERVICE_NAME]: serviceName
+    })
+  });
+
+  let exporter;
+  if (EXPORTER === 'jaeger') {
+    exporter = new JaegerExporter();
+  } else {
+    exporter = new ZipkinExporter();
+  }
+  provider.addSpanProcessor(new SimpleSpanProcessor(exporter));
+
+  registerInstrumentations({
+    instrumentations: [
+      new KoaInstrumentation(),
+      new HttpInstrumentation(),
+    ],
+    tracerProvider: provider,
+  });
+
+  // Initialize the OpenTelemetry APIs to use the NodeTracerProvider bindings
+  provider.register();
+
+  return api.trace.getTracer(serviceName);
```

Comment:
tracer name is usually set to a **package name** ( as set in the `name` field in `package.json`) which in our case is `koa-example`

Reply:
Also here I took this behavior from here: https://github.com/open-telemetry/opentelemetry-js-contrib/pull/939/files, see tracer.ts file, line 50. this PR appears in the issue as a reference, so I thought this is ok they will be aligned.

Reply:
I think this is an overlooked mistake in express example PR. 
This is a minor issue but would love other opinions @open-telemetry/javascript-approvers